template/lambda-sqs-worker-cdk/infra/config.ts

```diff
@@ -33,9 +35,10 @@ export const configs: Record<Environment, Config> = {
       environment: {
         ENVIRONMENT: 'prod',
       },
+      SERVICE: '<%- serviceName %>',
```

Comment:
This will have to be in environment to be passed to the worker.

Could we also add VERSION but we're going to take in an environment variable which will come from our CI

https://github.com/seek-oss/skuba/tree/main/packages/skuba-dive#env

Could we also add this env var into the pipeline
https://github.com/seek-oss/skuba/blob/81780d930e8c2dd3f720f52223895580407af61d/template/lambda-sqs-worker/.buildkite/pipeline.yml#L50

Reply:
This is looking excellent

Reply:
Ah yep! I think I get you now., I think I'd lost sight of the original purpose of the PR and was trying to avoid editing it too much... I hope this is better.

I don't think I can run the snapshot actions but I updated the relevant snapshot manually